### PR TITLE
[Sass/de-de] Added German version for Sass

### PR DESCRIPTION
**Pull request by phbo85**

### CODE REVIEW

File: de-de/sass-de.html.markdown [hunk 1]
```diff
@@ -0,0 +1,450 @@
+---
+language: sass
+filename: learnsass.scss
```
Comment (vendethiel) [inline de-de/sass-de.html.markdown]: please add `-de` suffix to filename

File: de-de/sass-de.html.markdown [hunk 2]
```diff
@@ -0,0 +1,450 @@
...
+   und die benötigten Parameter übergibst. */
+body {
+  width: round(10.25px);
+}
+
+.footer {
+  background-color: fade_out(#000000, 0.25)
+}
+
+/* Kompliert: */
```
Comment (m90) [inline de-de/sass-de.html.markdown]: Minor typo, should be `Kompiliert`

File: de-de/sass-de.html.markdown [hunk 3]
```diff
@@ -0,0 +1,450 @@
...
+  width: 10px;
+}
+
+.footer {
+  background-color: rgba(0, 0, 0, 0.75);
+}
+
+/* Du kannst auch deine eigenen Funktionen definieren. Funktionen ähneln
+   mixins. Wenn du zwischen Funktionen und Mixins auswählen musst, denke
+   daran das mixins am besten zur Generierung von CSS eignen, während
```
Comment (m90) [inline de-de/sass-de.html.markdown]: Should be `dass`

File: de-de/sass-de.html.markdown [hunk 4]
```diff
@@ -0,0 +1,450 @@
...
+
+```
+
+## SASS oder Sass?
+Hast du dich jemals gefragt, ob Sass ein Akronym ist oder nicht? Hast du wahrscheinlich nicht, aber ich sage es dir trotzdem. Der Name der Sprache ist ein Wort, "Sass", und kein Akronym.
+Da die Leute durchgehend "SASS" geschrieben haben, hat der Ersteller der Sprache es scherzhaft "Syntactically Awesome StyleSheets" genannt.
+
+## Sass üben
+Wenn du mit Sass in deinem Browser spielen willst, schau dir [SassMeister](http://sassmeister.com/) an.
+Du kannst beide Syntaxen benutzen, gehe einfach in die Einstellungen und wähle entweder Sass oder SCSS.
```
Comment (m90) [inline de-de/sass-de.html.markdown]: `Syntaxen` sounds pretty "gestelzt" to me. Wouldn't `Dialekt` be better here? Not 100% sure though.

File: de-de/sass-de.html.markdown [hunk 5]
```diff
@@ -0,0 +1,450 @@
...
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+	left: 0;
+	right: 0;
+	background-color: #A3A4FF;
+}
+
+
+/* Du kannst mixins benutzen, um shorthand Eigenschaften zu erstellen. */
```
Comment (m90) [inline de-de/sass-de.html.markdown]: You are sometimes using `mixins` and sometimes `Mixins`. I'd go for `Mixins` everywhere.

**Comment by m90** [comment]
Apart from the minor nitpicks, the translation is of very good quality, +1 from a native speaker.

**Comment by vendethiel** [comment]
@phbo85 , any updates?:)

**Comment by phbo85** [comment]
I created a now pull request with the fixed errors and the suffix: https://github.com/adambard/learnxinyminutes-docs/pull/2291

**Pull request closed without merging**